openquake/ghm/create_homogenised_curves.py

```diff
@@ -309,14 +309,16 @@ def process_maps(contacts_shp, outpath, datafolder, sidx_fname, boundaries_shp,
                     tmpdf = copy.deepcopy(map_gdf[idx])
                     tmpdf = tmpdf.set_crs('epsg:4326')
                     tmpdf = gpd.sjoin(tmpdf, inland_df, how='inner',
-                                      predicate='intersects')
-
-                    p_tmpdf = tmpdf.to_crs('epsg:3857')
+                                       predicate='intersects')
+                    
+                    ##keeping the crs as epsg:4326.
+                    #p_tmpdf = tmpdf.to_crs('epsg:3857')
```

Comment:
@mmpagani the commented lines were added to address the warning about inconsistent CRS that may yield incorrect distances. however, they caused major issues in the buffer zones, so we continued to use the former code. do you want to solve this problem before we merge this branch into master?